discovery-handlers/onvif/src/discovery_handler.rs

```diff
@@ -520,6 +558,7 @@ mod tests {
             }),
             scopes: None,
             discovery_timeout_seconds: 1,
+            query_device_http: None,
```

Comment:
we probably need a test somewhere where query_device_http==!None

Reply:
Will that be to create a mockup REST device inventory and check the query result of a test device?